xSqlPs.psd1

```diff
@@ -3,7 +3,7 @@
 @{
 
 
-ModuleVersion = '1.1.3.1'
+ModuleVersion = '1.1.4'
```

Comment:
Per [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module), Don't update the module version